src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -82,50 +127,17 @@ export class HuiCardPicker extends LitElement {
     }
 
     return html`
+      <search-input
+        .filter=${this._filter}
+        autofocus=${false}
```

Comment:
If you remove it, it will be false for a boolean attr
```suggestion
```

Reply:
Of course :facepalm:  :smile: